### DISCUSSION TITLE
控制中心-电源-节能设置 ，亮度降低最小值，无法设置为0%，是否合理

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#4490] [General | 常规讨论]
![截图_dde-control-center_20230525091512](https://github.com/linuxdeepin/developer-center/assets/7598566/0884fc48-ede5-435c-9ca2-779e12382528)

如图，有些用户不希望屏幕亮度有变化，但当前至少也要降低10%  ，用 dconf-editor 调整，普通用户又不一定会

所以，建议最小值从0% 开始

欢迎讨论

**Comment by BLumia** [comment]
cc @meiyixiang
